docs/dev_sliding_dot_product.ipynb

```diff
@@ -0,0 +1,981 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.    # I swapped the params of <code>fft</code> function to make its signature similar to <code>scipy.ftt.ftt</code>
</pre><p>This is an old comment. Remove it. Checked the other versions and fix them as well if needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/939/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>